sound/soc/sof/sof-client-probes-ipc3.c

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: GPL-2.0-only
+//
+// Copyright(c) 2019-2022 Intel Corporation. All rights reserved.
+//
+// Author: Jyri Sarha <jyri.sarha@intel.com>
```

Comment:
I would have probably kept the original author, this is just moving the code.

Reply:
Sure.